constns.py

```diff
@@ -0,0 +1,14 @@
+import threading
+import random
+
+counter = 0

```

Comment:
Consts are Static Variables, that means you dont change them.
If you want file with many changeable variables, rename it to variables.py.
Although it is not recommended, you should place the variables inside the module and not in different one.